core/src/com/projectkorra/projectkorra/airbending/Suffocate.java

```diff
@@ -107,21 +107,21 @@ public Suffocate(final Player player) {
 			}
 		} else {
 			List<Entity> entities = new ArrayList<Entity>();
-			for (int i = 0; i < 6; i++) {
+			for (int i = 0; i < range; i++) {
 				final Location location = GeneralMethods.getTargetedLocation(player, i, getTransparentMaterials());
 				entities = GeneralMethods.getEntitiesAroundPoint(location, .5);
-				if (entities.contains(player)) {
-					entities.remove(player);
-				}
-				if (entities != null && !entities.isEmpty() && !entities.contains(player)) {
+
+				entities.remove(player);
+
+				if (!entities.isEmpty() && !entities.contains(player)) {
```

Comment:
Is there a reason for the `!entities.contains(player)` with player being removed right before it? It should always return true